test/unittests/test_dd_synthesis.cpp

```diff
@@ -0,0 +1,248 @@
+#include "algorithms/synthesis/dd_synthesis.hpp"
+#include "algorithms/synthesis/encoding.hpp"
+#include "core/io/pla_parser.hpp"
+#include "dd/FunctionalityConstruction.hpp"
+
+#include "gtest/gtest.h"
+
+using namespace dd::literals;
+using namespace syrec;
+
+class TestDDSynth: public testing::Test {
+protected:
+    TruthTable                     tt{};
+    std::string                    testCircuitsDir = "./circuits/";
+    std::unique_ptr<dd::Package<>> ddSynth         = std::make_unique<dd::Package<>>(6U);
+};
+
+TEST_F(TestDDSynth, CUSTOMTT1) {
```

Comment:
Please use appropriate names for these test cases and also for all the files. I have no idea what `customtt1` should be.

Reply:
done 👍🏻